app/assets/javascripts/Components/Result/feedback_file_panel.jsx

```diff
@@ -46,6 +46,10 @@ export class FeedbackFilePanel extends React.Component {
         {feedback_file_id: this.state.selectedFile}
       );
       file_obj = this.props.feedbackFiles.find(file => file.id === this.state.selectedFile);
+
+      if (!file_obj) {
+        file_obj = this.props.feedbackFiles[0];
```

Comment:
Since this is in response to change results, I'm guessing `this.props.feedbackFiles` could be empty.

But also, I think a better approach would be to use a React lifecycle hook to update `this.state.selectedFile` when `this.props.feedbackFiles` changes (that keeps the props + state in sync, before the render).